build-tools/packages/build-cli/src/commands/vnext/check/latestVersions.ts

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { WorkspaceName } from "@fluid-tools/build-infrastructure";
+import { isInternalVersionScheme } from "@fluid-tools/version-tools";
+import * as semver from "semver";
+import { packageOrReleaseGroupArg, semverArg } from "../../../args.js";
+import {
+	BaseCommandWithBuildProject,
+	getAllPackageVersions,
+	sortVersions,
+} from "../../../library/index.js";
+
+type MajorVersion = number;
+
+export default class LatestVersionsCommand extends BaseCommandWithBuildProject<
+	typeof LatestVersionsCommand
+> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static readonly args = {
+		version: semverArg({
+			required: true,
+			description:
+				"The version to check. When running in CI, this value corresponds to the pipeline trigger branch.",
+		}),
+		package_or_release_group: packageOrReleaseGroupArg({ required: true }),
+	} as const;
+
+	public async run(): Promise<void> {
+		const { args } = this;
+		const repo = this.getBuildProject();
+		const rgOrPackage = repo.workspaces.get(args.package_or_release_group as WorkspaceName);
+
+		const versionInput = this.args.version;
```

Comment:
You extracted args from this earlier - so either use just args or use this.args everywhere.